<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>936</width>
    <height>702</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../build-untitled-Desktop_Qt_5_9_0_MSVC2015_32bit-Debug/icon/main_title.png</normaloff>../build-untitled-Desktop_Qt_5_9_0_MSVC2015_32bit-Debug/icon/main_title.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="verticalLayoutWidget_5">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>571</width>
      <height>631</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_6" stretch="1,3,3,3">
     <item>
      <widget class="QGroupBox" name="groupBox_6">
       <property name="styleSheet">
        <string notr="true">QGroupBox {
	border: 1px solid #00BFFF;
	border-radius:10px
}</string>
       </property>
       <property name="title">
        <string>听力名称</string>
       </property>
       <widget class="QLabel" name="label_info">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>20</y>
          <width>561</width>
          <height>19</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QGroupBox" name="groupBox_7">
         <property name="styleSheet">
          <string notr="true">QGroupBox {
	border: 1px solid #00BFFF;
	border-radius:10px
}</string>
         </property>
         <property name="title">
          <string>播放控制</string>
         </property>
         <widget class="QWidget" name="verticalLayoutWidget">
          <property name="geometry">
           <rect>
            <x>20</x>
            <y>30</y>
            <width>521</width>
            <height>138</height>
           </rect>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <widget class="QSlider" name="horizon_music">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_4">
             <item>
              <widget class="QPushButton" name="button_play">
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>30</height>
                </size>
               </property>
               <property name="styleSheet">
                <string notr="true">QPushButton{
	background-color:rgba(100,225,100,30);
	border-style:outset;
	border-width:4px;           
	border-radius:10px;
	border-color:rgba(255,255,255,30);
	font:Time New Roams bold 10px;
	color:rgba(0,0,0,100);
	padding:6px;
}    
QPushButton:pressed{     
	background-color:rgba(100,255,100,200);    
	border-color:rgba(255,255,255,30);   
	border-style:inset;    
	color:rgba(0,0,0,100);    
}    

QPushButton:hover{    
	background-color:rgba(100,255,100,100);    
	border-color:rgba(255,255,255,200);    
	color:rgba(0,0,0,200);    
}</string>
               </property>
               <property name="text">
                <string>Play</string>
               </property>
               <property name="shortcut">
                <string>Space</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="button_stop">
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>30</height>
                </size>
               </property>
               <property name="styleSheet">
                <string notr="true">QPushButton{
	background-color:rgba(100,225,100,30);
	border-style:outset;
	border-width:4px;           
	border-radius:10px;
	border-color:rgba(255,255,255,30);
	font:Time New Roams bold 10px;
	color:rgba(0,0,0,100);
	padding:6px;
}    
QPushButton:pressed{     
	background-color:rgba(100,255,100,200);    
	border-color:rgba(255,255,255,30);   
	border-style:inset;    
	color:rgba(0,0,0,100);    
}    

QPushButton:hover{    
	background-color:rgba(100,255,100,100);    
	border-color:rgba(255,255,255,200);    
	color:rgba(0,0,0,200);    
}</string>
               </property>
               <property name="text">
                <string>Stop</string>
               </property>
              </widget>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_5">
               <item>
                <widget class="QLabel" name="label_elapse_time">
                 <property name="text">
                  <string/>
                 </property>
                 <property name="alignment">
                  <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="label_totle_time">
                 <property name="text">
                  <string/>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <widget class="QPushButton" name="button_back">
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>30</height>
                </size>
               </property>
               <property name="styleSheet">
                <string notr="true">QPushButton{
	background-color:rgba(100,225,100,30);
	border-style:outset;
	border-width:4px;           
	border-radius:10px;
	border-color:rgba(255,255,255,30);
	font:Time New Roams bold 10px;
	color:rgba(0,0,0,100);
	padding:6px;
}    
QPushButton:pressed{     
	background-color:rgba(100,255,100,200);    
	border-color:rgba(255,255,255,30);   
	border-style:inset;    
	color:rgba(0,0,0,100);    
}    

QPushButton:hover{    
	background-color:rgba(100,255,100,100);    
	border-color:rgba(255,255,255,200);    
	color:rgba(0,0,0,200);    
}</string>
               </property>
               <property name="text">
                <string>&lt;&lt;(3s)</string>
               </property>
               <property name="shortcut">
                <string>Left</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="button_forward">
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>30</height>
                </size>
               </property>
               <property name="styleSheet">
                <string notr="true">QPushButton{
	background-color:rgba(100,225,100,30);
	border-style:outset;
	border-width:4px;           
	border-radius:10px;
	border-color:rgba(255,255,255,30);
	font:Time New Roams bold 10px;
	color:rgba(0,0,0,100);
	padding:6px;
}    
QPushButton:pressed{     
	background-color:rgba(100,255,100,200);    
	border-color:rgba(255,255,255,30);   
	border-style:inset;    
	color:rgba(0,0,0,100);    
}    

QPushButton:hover{    
	background-color:rgba(100,255,100,100);    
	border-color:rgba(255,255,255,200);    
	color:rgba(0,0,0,200);    
}</string>
               </property>
               <property name="text">
                <string>&gt;&gt;(3s)</string>
               </property>
               <property name="shortcut">
                <string>Right</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout">
             <item>
              <widget class="QPushButton" name="button_set_start">
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>30</height>
                </size>
               </property>
               <property name="styleSheet">
                <string notr="true">QPushButton{
	background-color:rgba(100,225,100,30);
	border-style:outset;
	border-width:4px;           
	border-radius:10px;
	border-color:rgba(255,255,255,30);
	font:Time New Roams bold 10px;
	color:rgba(0,0,0,100);
	padding:6px;
}    
QPushButton:pressed{     
	background-color:rgba(100,255,100,200);    
	border-color:rgba(255,255,255,30);   
	border-style:inset;    
	color:rgba(0,0,0,100);    
}    

QPushButton:hover{    
	background-color:rgba(100,255,100,100);    
	border-color:rgba(255,255,255,200);    
	color:rgba(0,0,0,200);    
}</string>
               </property>
               <property name="text">
                <string>Set Start Point</string>
               </property>
               <property name="shortcut">
                <string>[</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_beg_point">
               <property name="styleSheet">
                <string notr="true">QLabel  {
	border:1px solid yellow;
}</string>
               </property>
               <property name="text">
                <string/>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="button_clear_beg_point">
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>30</height>
                </size>
               </property>
               <property name="styleSheet">
                <string notr="true">QPushButton{
	background-color:rgba(100,225,100,30);
	border-style:outset;
	border-width:4px;           
	border-radius:10px;
	border-color:rgba(255,255,255,30);
	font:Time New Roams bold 10px;
	color:rgba(0,0,0,100);
	padding:6px;
}    
QPushButton:pressed{     
	background-color:rgba(100,255,100,200);    
	border-color:rgba(255,255,255,30);   
	border-style:inset;    
	color:rgba(0,0,0,100);    
}    

QPushButton:hover{    
	background-color:rgba(100,255,100,100);    
	border-color:rgba(255,255,255,200);    
	color:rgba(0,0,0,200);    
}</string>
               </property>
               <property name="text">
                <string>Clear</string>
               </property>
               <property name="shortcut">
                <string>{</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <item>
              <widget class="QPushButton" name="button_set_end">
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>30</height>
                </size>
               </property>
               <property name="styleSheet">
                <string notr="true">QPushButton{
	background-color:rgba(100,225,100,30);
	border-style:outset;
	border-width:4px;           
	border-radius:10px;
	border-color:rgba(255,255,255,30);
	font:Time New Roams bold 10px;
	color:rgba(0,0,0,100);
	padding:6px;
}    
QPushButton:pressed{     
	background-color:rgba(100,255,100,200);    
	border-color:rgba(255,255,255,30);   
	border-style:inset;    
	color:rgba(0,0,0,100);    
}    

QPushButton:hover{    
	background-color:rgba(100,255,100,100);    
	border-color:rgba(255,255,255,200);    
	color:rgba(0,0,0,200);    
}</string>
               </property>
               <property name="text">
                <string>Set End Point</string>
               </property>
               <property name="shortcut">
                <string>]</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_end_point">
               <property name="styleSheet">
                <string notr="true">QLabel  {
	border:1px solid yellow;
}</string>
               </property>
               <property name="text">
                <string/>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="button_clear_end_point">
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>30</height>
                </size>
               </property>
               <property name="styleSheet">
                <string notr="true">QPushButton{
	background-color:rgba(100,225,100,30);
	border-style:outset;
	border-width:4px;           
	border-radius:10px;
	border-color:rgba(255,255,255,30);
	font:Time New Roams bold 10px;
	color:rgba(0,0,0,100);
	padding:6px;
}    
QPushButton:pressed{     
	background-color:rgba(100,255,100,200);    
	border-color:rgba(255,255,255,30);   
	border-style:inset;    
	color:rgba(0,0,0,100);    
}    

QPushButton:hover{    
	background-color:rgba(100,255,100,100);    
	border-color:rgba(255,255,255,200);    
	color:rgba(0,0,0,200);    
}</string>
               </property>
               <property name="text">
                <string>Clear</string>
               </property>
               <property name="shortcut">
                <string>}</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QGroupBox" name="groupBox_2">
         <property name="styleSheet">
          <string notr="true">QGroupBox {
	border: 1px solid #00BFFF;
	border-radius:10px
}</string>
         </property>
         <property name="title">
          <string>听写内容</string>
         </property>
         <widget class="QPlainTextEdit" name="text_type">
          <property name="geometry">
           <rect>
            <x>20</x>
            <y>20</y>
            <width>521</width>
            <height>141</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <family>Times New Roman</family>
            <pointsize>16</pointsize>
           </font>
          </property>
         </widget>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_6">
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="styleSheet">
          <string notr="true">QGroupBox {
	border: 1px solid #00BFFF;
	border-radius:10px
}</string>
         </property>
         <property name="title">
          <string>备注</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
         </property>
         <widget class="QPlainTextEdit" name="text_note">
          <property name="geometry">
           <rect>
            <x>20</x>
            <y>20</y>
            <width>241</width>
            <height>141</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <family>Times New Roman</family>
            <pointsize>16</pointsize>
           </font>
          </property>
         </widget>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_5">
         <property name="styleSheet">
          <string notr="true">QGroupBox {
	border: 1px solid #00BFFF;
	border-radius:10px
}</string>
         </property>
         <property name="title">
          <string>标记设置</string>
         </property>
         <widget class="QComboBox" name="combox_level">
          <property name="geometry">
           <rect>
            <x>100</x>
            <y>40</y>
            <width>71</width>
            <height>31</height>
           </rect>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>25</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Times New Roman</family>
            <pointsize>12</pointsize>
           </font>
          </property>
         </widget>
         <widget class="QComboBox" name="combox_sort">
          <property name="geometry">
           <rect>
            <x>50</x>
            <y>80</y>
            <width>171</width>
            <height>31</height>
           </rect>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>25</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Times New Roman</family>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QCombox_sort{
	border: 1px;solid;#999999;
	border-radius: 5px;
}</string>
          </property>
          <property name="iconSize">
           <size>
            <width>16</width>
            <height>16</height>
           </size>
          </property>
         </widget>
         <widget class="QPushButton" name="button_delete">
          <property name="geometry">
           <rect>
            <x>150</x>
            <y>130</y>
            <width>71</width>
            <height>31</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton{
	background-color:rgba(100,225,100,30);
	border-style:outset;
	border-width:4px;           
	border-radius:10px;
	border-color:rgba(255,255,255,30);
	font:Time New Roams bold 10px;
	color:rgba(0,0,0,100);
	padding:6px;
}    
QPushButton:pressed{     
	background-color:rgba(255,0,0,200);    
	border-color:rgba(255,255,255,30);   
	border-style:inset;    
	color:rgba(0,0,0,100);    
}    

QPushButton:hover{    
	background-color:rgba(255,0,0,100);    
	border-color:rgba(255,255,255,200);    
	color:rgba(0,0,0,200);    
}</string>
          </property>
          <property name="text">
           <string>Delete</string>
          </property>
          <property name="shortcut">
           <string>Del</string>
          </property>
         </widget>
         <widget class="QPushButton" name="button_save">
          <property name="geometry">
           <rect>
            <x>40</x>
            <y>130</y>
            <width>81</width>
            <height>31</height>
           </rect>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>23</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton{
	background-color:rgba(100,225,100,30);
	border-style:outset;
	border-width:4px;           
	border-radius:10px;
	border-color:rgba(255,255,255,30);
	font:Time New Roams bold 10px;
	color:rgba(0,0,0,100);
	padding:6px;
}    
QPushButton:pressed{     
	background-color:rgba(100,255,100,200);    
	border-color:rgba(255,255,255,30);   
	border-style:inset;    
	color:rgba(0,0,0,100);    
}    

QPushButton:hover{    
	background-color:rgba(100,255,100,100);    
	border-color:rgba(255,255,255,200);    
	color:rgba(0,0,0,200);    
}</string>
          </property>
          <property name="text">
           <string>Save</string>
          </property>
          <property name="shortcut">
           <string>Return</string>
          </property>
         </widget>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_4">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>10</y>
      <width>321</width>
      <height>631</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_5">
     <item>
      <widget class="QGroupBox" name="groupBox_3">
       <property name="styleSheet">
        <string notr="true">QGroupBox {
	border: 1px solid #00BFFF;
	border-radius:10px
}</string>
       </property>
       <property name="title">
        <string>标记</string>
       </property>
       <widget class="QListWidget" name="listWidget">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>20</y>
          <width>301</width>
          <height>501</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Times New Roman</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="contextMenuPolicy">
         <enum>Qt::CustomContextMenu</enum>
        </property>
        <property name="uniformItemSizes">
         <bool>false</bool>
        </property>
       </widget>
       <widget class="QGroupBox" name="groupBox_4">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>540</y>
          <width>301</width>
          <height>71</height>
         </rect>
        </property>
        <property name="title">
         <string>排序</string>
        </property>
        <widget class="QComboBox" name="combox_sort_level">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>20</y>
           <width>81</width>
           <height>31</height>
          </rect>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>25</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Times New Roman</family>
           <pointsize>12</pointsize>
          </font>
         </property>
        </widget>
        <widget class="QComboBox" name="combox_sort_sort">
         <property name="geometry">
          <rect>
           <x>100</x>
           <y>20</y>
           <width>191</width>
           <height>31</height>
          </rect>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>25</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Times New Roman</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="iconSize">
          <size>
           <width>16</width>
           <height>16</height>
          </size>
         </property>
        </widget>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>1060</x>
      <y>130</y>
      <width>160</width>
      <height>84</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_7"/>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>936</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>文件</string>
    </property>
    <addaction name="syncMarks"/>
    <addaction name="onlineFile"/>
    <addaction name="actionopen"/>
   </widget>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>关于</string>
    </property>
    <addaction name="howToUse"/>
    <addaction name="about"/>
    <addaction name="exit"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menu"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionopen">
   <property name="icon">
    <iconset>
     <normaloff>../build-untitled-Desktop_Qt_5_9_0_MSVC2015_32bit-Release/icon/file_local.png</normaloff>../build-untitled-Desktop_Qt_5_9_0_MSVC2015_32bit-Release/icon/file_local.png</iconset>
   </property>
   <property name="text">
    <string>打开本地文件</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="howToUse">
   <property name="icon">
    <iconset>
     <normaloff>../build-untitled-Desktop_Qt_5_9_0_MSVC2015_32bit-Debug/icon/how.png</normaloff>../build-untitled-Desktop_Qt_5_9_0_MSVC2015_32bit-Debug/icon/how.png</iconset>
   </property>
   <property name="text">
    <string>使用说明</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="about">
   <property name="icon">
    <iconset>
     <normaloff>../build-untitled-Desktop_Qt_5_9_0_MSVC2015_32bit-Debug/icon/about.png</normaloff>../build-untitled-Desktop_Qt_5_9_0_MSVC2015_32bit-Debug/icon/about.png</iconset>
   </property>
   <property name="text">
    <string>关于</string>
   </property>
  </action>
  <action name="exit">
   <property name="icon">
    <iconset>
     <normaloff>../build-untitled-Desktop_Qt_5_9_0_MSVC2015_32bit-Debug/icon/exit.png</normaloff>../build-untitled-Desktop_Qt_5_9_0_MSVC2015_32bit-Debug/icon/exit.png</iconset>
   </property>
   <property name="text">
    <string>退出</string>
   </property>
  </action>
  <action name="onlineFile">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../build-untitled-Desktop_Qt_5_9_0_MSVC2015_32bit-Release/icon/file_online.png</normaloff>../build-untitled-Desktop_Qt_5_9_0_MSVC2015_32bit-Release/icon/file_online.png</iconset>
   </property>
   <property name="text">
    <string>在线下载音频</string>
   </property>
  </action>
  <action name="syncMarks">
   <property name="text">
    <string>同步标记</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
